terra-resource-janitor-client/build.gradle

```diff
@@ -20,6 +20,8 @@ dependencies {
     implementation group: 'org.glassfish.jersey.core', name: 'jersey-client', version: "${jersey}"
     implementation group: 'org.glassfish.jersey.media', name: 'jersey-media-json-jackson', version: "${jersey}"
     implementation group: 'org.glassfish.jersey.media', name: 'jersey-media-multipart', version: "${jersey}"
+    implementation "org.glassfish.jersey.inject:jersey-hk2:${jersey}"
```

Comment:
what do we need this for?

Reply:
Ah, I think I can remove this one. I got some jersey binding exceptions in CRL when uses janitor calls. I saw data repo had that and it works. But I also added the same dependency in CRL, and it solves that problem. Let me delete this. 
Same issue as this: https://stackoverflow.com/questions/44088493/jersey-stopped-working-with-injectionmanagerfactory-not-found